integration/src/extension.rs

```diff
@@ -0,0 +1,126 @@
+use std::fs;
+use std::path::{Path, PathBuf};
+
+use assert_cmd::assert::Assert;
+use assert_cmd::Command;
+use tempfile::TempDir;
+
+const HEADER: &str = "import { PhylumApi } from 'phylum';";
+
+/// Test Phylum API functions.
+#[cfg(test)]
+pub fn api(config: &Path) {
```

Comment:
This file looks like a duplicate of `end_to_end/extension.rs`, is this intended?

Reply:
No.